packages/x-components/tests/e2e/cucumber/popular-searches/popular-searches.spec.ts

```diff
@@ -46,7 +59,7 @@ When('popular search number {int} is clicked', (popularSearchItem: number) => {
 });
 
 And(
-  'the clicked popular search is removed from Popular Searches if {boolean} is true',
+  'the clicked popular search is removed from Popular Searches if hideSessionQueries = {boolean} is true',
```

Comment:
Please fix this error

![image](https://user-images.githubusercontent.com/3502075/129346019-a656aa47-9813-4f5a-baf5-1af6d09f1aae.png)